src/components/permissions/loginHelpers.ts

```diff
@@ -68,11 +68,14 @@ const getUserInfo = async (userOrgParameters: UserOrgParameters) => {
 
   const returnSessionId = sessionId ?? nanoid(16)
 
-  const isAdmin = !!templatePermissionRows.find((row) => !!row?.isAdmin)
-
   const managementPrefName =
     config?.systemManagerPermissionName || config.defaultSystemManagerPermissionName
-  const isManager = !!templatePermissionRows.includes(managementPrefName)
```

Comment:
This expression didn't work at all!

I've combined `isManager` and `isAdmin` into a single database call that is independent of template permissions. (The previous way, if a user was Admin, but there were no templates that required Admin permission, then `isAdmin` would return `false`. As for `isManager`, well...🙄)